src/types/config.d.mts

```diff
@@ -61,7 +61,7 @@ declare global {
    * }
    * ```
    */
-  //eslint-disable-next-line @typescript-eslint/no-empty-interface
+  //eslint-disable-next-line @typescript-eslint/no-empty-interface, @typescript-eslint/no-empty-object-type
```

Comment:
Nit; this isn't even your fault but this line is `//eslint-disable-next-line` and I'd prefer `// eslint-disable-next-line` for consistency. I won't note it everywhere because fixing this should be as simple as a single find and replace.